Tests/BacktraceMetricsTest.swift

```diff
@@ -16,35 +16,31 @@ final class BacktraceMetricsTests: QuickSpec {
 
             let summedEventName = "view-changed"
             let uniqueEventName = "guid"
-
-            it("clears the summed event after enabling and sending") {
-                let metrics = BacktraceMetrics(api: backtraceApi)
-                metrics.enable(settings: BacktraceMetricsSettings())
-
-                // Allow default events to be "sent" out
-                expect { metrics.count }.toEventually(equal(1), timeout: .seconds(5), pollInterval: .milliseconds(100))
-            }
-
-            it("can add and store summed event") {
-                let metrics = BacktraceMetrics(api: backtraceApi)
-                metrics.enable(settings: BacktraceMetricsSettings())
-
-                // Allow default events to be "sent" out
-                expect { metrics.count }.toEventually(equal(1), timeout: .seconds(5), pollInterval: .milliseconds(100))
-
-                metrics.addSummedEvent(name: summedEventName)
-                expect { metrics.count }.to(equal(2))
-            }
-
-            it("can add and store unique event") {
-                let metrics = BacktraceMetrics(api: backtraceApi)
-                metrics.enable(settings: BacktraceMetricsSettings())
-
-                // Allow default events to be "sent" out
-                expect { metrics.count }.toEventually(equal(1), timeout: .seconds(5), pollInterval: .milliseconds(100))
-
-                metrics.addUniqueEvent(name: uniqueEventName)
-                expect { metrics.count }.to(equal(2))
+            
+            var metrics: BacktraceMetrics?
+            
+            context("Events operation") {
+                
+                beforeEach {
+                    metrics = BacktraceMetrics(api: backtraceApi)
+                    metrics?.enable(settings: BacktraceMetricsSettings())
+                    metrics?.clearSummedEvents()
+                }
+                
+                it("clears the summed event after enabling and sending") {
+                    // Allow default events to be "sent" out
+                    expect { metrics?.count }.toEventually(equal(1))
```

Comment:
why 1? I think its becuase the unique event is still there. Maybe we can do better and get all events and filter them by checking the event type? Maybe it is just enough if we move 1 to variable and call it expectedNumberOfEvents

Reply:
I think we should verify it the number of summed events is 1. metrics.count will return number of events in the queue (unique + summed)